AuthorService.go

```diff
@@ -7,15 +7,15 @@ type AuthorService struct {
 
```

Comment:
Can't comment it on the spot, but note line 3:
 // AuthorService extracts and transforms the author taxonomy into a **suggestion**